package.json

```diff
@@ -33,12 +34,13 @@
     "typescript": "^4.2.3"
   },
   "scripts": {
-    "install": "cd packages/docs && npm ci",
+    "install": "test -n $SKIP_DOCS_INSTALL || (cd packages/docs && npm ci)",
```

Comment:
I'm thinking we might want to remove the docs dependencies out of this completely and only install them on demand.
Just not sure about the local dev experience.